.github/workflows/build-pip-wheels.yml

```diff
@@ -121,7 +121,7 @@ jobs:
         uses: actions/checkout@v4
         with:
           fetch-depth: 0
-          submodules: "recursive"
+          submodules: recursive
```

Comment:
why removed ""? a sign for merging child branches?

Reply:
So, all of the checkouts are recursive, i.e. they will descend into submodules (recursively!) and fetch those in addition. However, in all other instances we don't use `"`, so I removed them here, too.